framework/src/outputs/ConsoleUtils.C

```diff
@@ -96,8 +96,14 @@ outputMeshInformation(FEProblemBase & problem, bool verbose)
         << std::setw(console_field_width)
         << "  Num Partitions: " << static_cast<std::size_t>(mesh.n_partitions()) << '\n';
     if (problem.n_processors() > 1)
-      oss << std::setw(console_field_width) << "  Partitioner: " << moose_mesh.partitionerName()
-          << (moose_mesh.isPartitionerForced() ? " (forced) " : "") << '\n';
+    {
+      oss << std::setw(console_field_width) << "  Partitioner: ";
+      if (moose_mesh.isCustomPartitionerRequested())
+        oss << "custom";
```

Comment:
I don't see the point of this. The source of the "partitionerName" is `MooseMesh::partitionerName` or arguably the _partitioner instance itself. Perhaps we can look into fixing it in the right spot?

Reply:
Problem is that _partitioner may not have a name and its name is not like the type we put in here for other cases.

Reply:
Well perhaps we should fix that... Too much scope for this PR? I could put @fdkong on this. The Partitioner base class has a name so we really should be pulling it from there (encapsulation). Even putting this logic into the other method would be a little better in case other methods access that method directly.

Reply:
`libMesh::Partitioner` does not have a name. I will drop this commit because I really want the other one merged.